gnomad/utils/transcript_annotation.py

```diff
@@ -36,10 +36,9 @@ def summarize_transcript_expression(
         quantification. Default is 'transcript_tpm'.
     :param tissue_expr: Column expression indicating tissue type. Default is 'tissue'.
     :param summary_agg_func: Optional aggregation function to use to summarize the
-        transcript expression quantification by tissue. Example: `hl.agg.mean`. Default
+        transcript expression quantification by tissue. Example: `hl.mean`. Default
```

Comment:
It needs to use an aggregation function. There is no hl.agg.median, so that is why the `hl.median(hl.agg.collect(x))` is used below. We could also switch to use `hl.agg.approx_median` as the default, but it's a small dataset which is why the collect here wasn't that big of an issue.

```suggestion
        transcript expression quantification by tissue. Example: `hl.agg.mean`. Default
```